buildpipeline/windows.groovy

```diff
@@ -78,6 +78,8 @@ simpleNode('Windows_NT','latest') {
                     }
                 } else if (params.TGroup == 'uap') {
                     targetHelixQueues = ['Windows.10.Amd64.ClientRS2.Open']
+                } else if (params.TGroup == 'netfx') {
+                    targetHelixQueues = ['Windows.10.Amd64.ClientRS3']
```

Comment:
Separately are you planning to chagne the line a little higher up so netcoreapp tests run on RS3? (Or rather RS4)

Reply:
RS4 we are waiting for them to be created, and RS3 it seems like we need an Open queue. So I'll figure it out with @MattGal